src/PolygonClipping.jl

```diff
@@ -3,19 +3,19 @@ module PolygonClipping
 import Base.show
 import Base.push!
 import Base.length
-using ImmutableArrays
+import StaticArrays.ImmutableArrays.Vector2
```

Comment:
Hah, it's a long time since I saw code using `ImmutableArrays`.  Good to see that the `StaticArrays` compatibility interface still works ok here :-)